javascript/operations.js

```diff
@@ -452,8 +466,8 @@ export default {
     operate(operation, () => {
       const { message, level } = operation
       level && ['warn', 'info', 'error'].includes(level)
-        ? console[level](message || '')
-        : console.log(message || '')
+        ? console[level](message != null ? message : '')
+        : console.log(message != null ? message : '')
```

Comment:
Technically `console.log` can accept any data type. I wonder if we should omit type coercion for this one.

Reply:
I agree:
`cable_ready.console_log(message: nil})` will be communicating better that it received `nil` rather than a empty string